extensions/amp-story/0.1/sources.js

```diff
@@ -48,6 +50,31 @@ export class Sources {
       element.setAttribute('src', this.srcAttr_);
     }
 
+    // Wait for "loadedmetadata" before adding tracks
+    // Firefox issue workaround
+    const addTracksHandler = () => {
```

Comment:
Would be good to move this up outside of the class so that it's only declared once, rather than on every invocation of `applyToElement(...)`

Reply:
This function uses `this.trackEls` to apply tracks, but inner `forEach` call can be moved outside to its own method in the class